tests/check-legal-aid/nightwatch.conf.js

```diff
@@ -0,0 +1,41 @@
+var puppeteer = require("puppeteer");
+
+module.exports = {
+  src_folders: "test/specs",
+  output_folder: "reports",
+  custom_assertions_path: "test/custom_assertions",
+  custom_commands_path: "test/commands",
+  globals_path: "test/globals/local.js",
+
+  selenium: {
+    start_process: false
+  },
+
+  test_settings: {
+    default: {
+      launch_url: "http://localhost:5000",
+      selenium_port: 9515,
+      selenium_host: "localhost",
+      default_path_prefix: "",
+      skip_testcases_on_fail: false,
+
+      desiredCapabilities: {
+        browserName: "chrome",
+        chromeOptions: {
+          args: ["--no-sandbox"]
+        },
+        acceptSslCerts: true
+      }
+    },
+
+    docker: {
+      launch_url: "http://cla_public",
```

Comment:
In found in my earlier projects that separate configurations tend to slowly erode to being broken as often it's only the most used one that was maintained.

What do you think about dropping `docker` and making `default` configured by passing configuration in from the execution context (eg. `launch_url: process.env.CLA_PUBLIC_URL || "http://localhost:5000"`)?

The intent would be to make the tests unaware of what contexts they need to run in. We could configure headless/not headless similarly, if necessary.

Reply:
Great feedback!